### DISCUSSION TITLE
Just wanted to say Thank You.

### DISCUSSION THREAD

**Discussion by nrthbound** [seerge/g-helper#2694] [General]
When I got this laptop, (the 2021 model, basically fully specced out but it was like... $1200 off) I was super pumped. Then I got it, played on it, loved it, until I closed the lid to put it to sleep. Few hours later, come back and the battery is like... 20% lower than it was. What? Why? That makes no sense. Now, I've been a PC Builder for 30 years, and I know them inside and out but I could not for the life of me solve this stupid problem. And the way Windows 11 handles iGPU vs dGPU is a nightmare. I was super bummed, the laptop felt lower quality now because of how piss poorly the software managed it. 

Then, I stumbled upon G-Helper, and my goodness... Just like that (snaps fingers), all my issues were solved. A better, faster, more lightweight interface, and no more Armory Crate crap. Since getting G-Helper, I *love* this machine again. So, I wanted to thank the developers for taking their time to create it, but even more so, for sharing it with the rest of us. It's very kind, and for me, made me love my laptop again. So, thank you.

**Comment by seerge** [comment]
@nrthbound thanks for a kind feedback :)
